test/e2e/apimachinery/chunking.go

```diff
@@ -75,7 +75,12 @@ var _ = SIGDescribe("Servers with support for API chunking", func() {
 		})
 	})
 
-	ginkgo.It("should return chunks of results for list calls", func() {
+	/*
+		Release : v1.15
```

Comment:
Looks like you left this in from another PR?

Reply:
I've commited the chunking.go file without the changes from the other PR. Let's make sure to rebase before this is merged. cc @BobyMCbobs @hh